tests/tests/test-block-1.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from "chai";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+describeDevMoonbeam("Block 1", (context) => {
+  before("Setup: Create empty block", async () => {
+    await context.createBlock();
+  });
+
+  it("should be at block 1", async function () {
+    expect(await context.web3.eth.getBlockNumber()).to.equal(1);
+  });
+
+  it("should have valid timestamp after block production", async function () {
+    // Originally ,this test required the timestamp be in the last finve minutes.
+    // This requirement doesn't make sense when we forge timestamps in manual seal.
```

Comment:
We no longer forge timestamps so we could bring back that lower bound that was previously in Frontier if we want to.

https://github.com/paritytech/frontier/pull/170/files#diff-b097ce1343f1eba88e98eb8392348a53c4f2824ae98a75adbd02120570f5768aL63-L64

Reply:
Also, there is a typo in that comment. `finve` -> `five`